packages/shared/icons/sites.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Non breaking, but Tashi has a way to clean this up so it's smaller.
We might have to ask him at one point to do all icons in the minimized way.

Could you add a ticket so we don't forget?

Reply:
Would love to! Let me stir one up! 🚀